docs/index.rst

```diff
@@ -0,0 +1,51 @@
+
+Asterix: Simulate your High-Contrast Instruments
+=======================================================
+
+Asterix is a python based library for simulating high contrast instruments and testbeds with a strong
+focus on focal plane wavefront sensing and correction algorithms. 
+
+Asterix is publicaly available on `GitHub <https://github.com/johanmazoyer/Asterix>`_ and all contributions are welcome!
+
+The development of Asterix is led by Johan Mazoyer with major contributions Axel Potier, Iva Laginja, 
+and Raphaël Galicher from LESIA (Paris Observatory)
+
+.. toctree::
+    :maxdepth: 1
+    :caption: Getting Started
+
+    install
```

Comment:
I don't know where this shows up in the docs, but to save yourself some headaches in the future, I would delete the section "Dependencies". If you really want that info in the docs, I would link to the `environment.yml` or `setup.py` file. Otherwise you just have yet another place where you need to update the dependencies, which is easily forgotten.

Reply:
ok, right, I'll do it now